ez/ez_test.go

```diff
@@ -178,3 +180,83 @@ func TestYAMLConfigEnvFlagWithValidatingConfigInitiallyValid(t *testing.T) {
 		require.EqualError(t, err, "val1 201 > 200")
 	}
 }
+
+func TestJSONConfigEnvFlagWithNewConfigCallback(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	tmpDir := t.TempDir()
```

Comment:
Oh fancy! I didn't realize `t` had a `TempDir` method. That's very useful.

Reply:
Yeah, that was new in go 1.15.